system_tests/firewall_rule_tests.py

```diff
@@ -114,6 +116,20 @@ def test_0040_list_object_types(self):
         if object_res:
             self.assertTrue(len(object_res) > 0)
 
+    def test_0041_enabled_firewall_rules(self):
+        firewall_obj = FirewallRule(TestFirewallRules._client,
+                                    TestFirewallRules._name,
+                                    TestFirewallRules._rule_id)
+        result = firewall_obj.enabled_firewall_rules()
+        self.assertIsNotNone(result)
```

Comment:
Same for test cases , Write single method and test cases.
enabled_firewall_rules(True)
enabled_firewall_rules(False)